src/site/_data/newsletter/archive/2020/03/index.html

```diff
@@ -0,0 +1,770 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
```

Comment:
Eleventy's data folder will merge content into the data cascade. I don't think we should use it as a place to store static content. I think we'd want to instead put these files into src/site/content.

Would it make more sense to treat these like any other piece of content in the site and just have them be part of a regular eleventy collection? In other words we'd make them into .njk files and use a `- newsletter` tag on them.

Since we already have to manually check these files in, it means we could also just write down the important info like title, hero image, etc. in their frontmatter. Essentially all the stuff that newsletter.js is trying to dynamically do right now.

Instead of using srcdoc we would set the iframe src to the generated file.